docs/examples/frameworks/jax/flax-basic_example.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<p>When I clicked the "training example with pure JAX" link, it redirected me to raw output of the jupyter notebook. Is this intentional? Or maybe the reviewing tool fails here?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/4978/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Yes, the review tool can't handle these links. I checked them in the docs build.